src/projections/healpix.cpp

```diff
@@ -78,6 +79,12 @@ static double sign (double v) {
     return v > 0 ? 1 : (v < 0 ? -1 : 0);
 }
 
+static PJ_XY rotate(PJ_XY p, double a) {
```

Comment:
I suggest you use `angle` instead of `a` here. Makes it immediately clear what's going on